cmd/config/cmd.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"github.com/openshift/rosa/cmd/config/get"
+	"github.com/openshift/rosa/cmd/config/set"
+	"github.com/openshift/rosa/pkg/config"
+	"reflect"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func configVarDocs() (ret string) {
```

Comment:
@tirthct I think a comment explaining this function would be helpful.

Is there an update planned for the `Config` struct as I don't see any `doc` tags on it? Also a unit test would be awesome 👍

Reply:
Hey @robpblake, sorry I couldn't understand what you meant by this : Is there an update planned for the Config struct as I don't see any doc tags on it?
Also, this is mostly a documentation function, so adding a few comments here, but not sure if we can write tests for this function specifically.

Reply:
@tirthct The doc tags can be found here in OCM https://github.com/openshift-online/ocm-cli/blob/main/pkg/config/config.go#L43-L53

@robpblake is asking for an update to the ROSA config `doc:` tags since we are attempting to leverage them in this function.

Reply:
Added doc tags to config